src/main/java/dk/cs/aau/huppaal/presentations/LogTabPresentation.java

```diff
@@ -0,0 +1,175 @@
+                case SUBCOMPONENT -> {
+                    if(ref2.isEmpty())
+                        Log.addWarning(notValidLink);
+                    var component = SelectHelper.selectComponent(ref1);
+                    ref2.ifPresent(s -> SelectHelper.selectSubComponent(component, s));
+                }
+                case JORK -> {
+                    if(ref2.isEmpty())
+                        Log.addWarning(notValidLink);
+                    var component = SelectHelper.selectComponent(ref1);
+                    ref2.ifPresent(s -> SelectHelper.selectJork(component, s));
+                }
+                case EDGE -> {
+                    if(ref2.isEmpty())
+                        Log.addWarning(notValidLink);
+                    var component = SelectHelper.selectComponent(ref1);
+                    ref2.ifPresent(s -> SelectHelper.selectEdge(component, s));
+                }
+                case NAIL -> Log.addInfo("Edges and nails are not selectable yet");
+                // Try to open the link
+                default -> {
+                    var f = new File(ref);
+                    if(f.exists())
+                        Desktop.getDesktop().open(f);
+                    else
+                        Desktop.getDesktop().browse(new URI(ref));
+                }
+            }
+        } catch (Exception e) {
+            HUPPAAL.showToast(e.getMessage());
+            Log.addError(e.getMessage());
+        }
+    }
+
+    private void onLogAdded(Log log) {
+        if(!log.level().equals(controller.level))
+            return;
+        var logMessage = log.message();
+        var matcher = LogRegex.PATTERN.matcher(log.message());
+        var index = 0;
+        while(matcher.find()) {
+            logArea.appendText(logMessage.substring(index, matcher.start()));
+            index = matcher.end();
+            var q = LogLinkQuantifier.valueOf(matcher.group("quantifier").toUpperCase());
+            logArea.appendWithLink(matcher.group("display"), matcher.group(), q);
+        }
+        logArea.appendText(logMessage.substring(index) + "\n");
+    }
+
+    private void initializeLogArea(String textColor) {
+        logArea.setTextInsertionStyle(new TextStyle().updateTextColorWeb(textColor));
+        logArea.getStyleClass().add("log-text");
+        logArea.setEditable(false);
+        controller.logAreaInsertionPoint.getChildren().add(scrollPane);
+    }
+
+    private void initializeButtons() {
+        Tooltip.install(controller.clearLogsButton, new Tooltip("Clear log"));
+        controller.clearLogsButton.setOnMouseClicked(e -> {
+            Log.clearLogsForLevel(controller.level); // This is just to be polite to the logging framework, and your RAM
+            logArea.clear();
+        });
+
+        Tooltip.install(controller.wrapTextButton, new Tooltip("Toggle word wrap"));
+        controller.wrapTextButton.setOnMouseClicked(e -> {
+            wordwrap = !wordwrap;
+            setupWordWrap();
```

Comment:
Consider introducing the same logic used for `autoscroll` to enhance consistency

Reply:
Fixed in 4d37aa0a4e3a2e10370d22910649fe7b92be4f53